server/container/guice/jpa-guice/src/main/java/org/apache/james/JPAJamesServerMain.java

```diff
@@ -45,7 +46,8 @@ public static void main(String[] args) throws Exception {
         GuiceJamesServerImpl server = new GuiceJamesServerImpl()
                     .combineWith(jpaServerModule, 
                             new JMXServerModule(), 
-                            new LuceneSearchMailboxModule());
+                            new LuceneSearchMailboxModule())
+                    .overrideWith(new NoWebAdminFilterModule());
```

Comment:
I understand the jwt config stuff, but what is the link with JMAP, we were talking about WebAdmin, right ? WebAdmin has to be available for most backend, don't you think ? (I agree is out of barcamp scope, obviously)